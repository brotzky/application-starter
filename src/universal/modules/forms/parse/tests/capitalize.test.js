import capitalize from '../capitalize';

describe('[Form Field Parsing] capitalize', () => {
  describe('capitalize()', () => {
    it('should make the first letter capital of a string', () => {
      expect(capitalize('hello')).toMatchSnapshot();
    });
  });
});
