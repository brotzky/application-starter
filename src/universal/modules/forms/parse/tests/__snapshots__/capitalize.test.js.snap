// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Form Field Parsing] capitalize capitalize() should make the first letter capital of a string 1`] = `"Hello"`;
